packages/react-storage/src/components/StorageManager/utils/displayText.ts

```diff
@@ -29,7 +29,7 @@ export const defaultStorageManagerDisplayText = {
   dropFilesText: 'Drop files here or',
   pauseText: 'Pause',
   resumeText: 'Resume',
-  uploadSuccessfulText: 'Uploaded successfully',
+  uploadSuccessfulText: 'Uploaded',
```

Comment:
This is a minor tweak that I think makes the text fit both previously uploaded files (defaultFiles) and newly uploaded files.

Reply:
It's nice that changing default text isn't a breaking change anymore!